plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/cart-order-summary-subtotal/block.tsx

```diff
@@ -5,13 +5,29 @@ import { Subtotal, TotalsWrapper } from '@woocommerce/blocks-components';
 import { getCurrencyFromPriceResponse } from '@woocommerce/price-format';
 import { useStoreCart } from '@woocommerce/base-context/hooks';
 
-const Block = ( { className = '' }: { className?: string } ): JSX.Element => {
+export type BlockAttributes = {
+	className: string;
+	heading: string | null;
```

Comment:
```suggestion
	heading: string;
```

Shouldn't this element be always available? Either it's the custom heading of the merchant or the placeholder text.

Reply:
@nielslange no, this is the attribute as it's set in the block.json which has a default of null therefore can be string | null

Reply:
I assume this also depends on the question I raised in https://github.com/woocommerce/woocommerce/pull/53544#discussion_r1875964453, doesn't it?

Reply:
@nielslange I guess the point is from a review perspective the implementation of this type is not incorrect and I wanted to make that clear. if we change the code then yes the block def will change but what does the UX change have to do with that here? If I change the code then naturally this type definition will change.

Reply:
@nielslange I don't want to split hairs but the implementation of the type was correct. We implement types as we expect the type to be. When we change the UX we will change the code because the block.json attribute might have default to string. In that case yes we'll change the definition 😛 but we don't need to go over this now. We'll change the implementation, the code will change, then you'll re-review.